client/my-sites/customer-home/cards/education/store/index.jsx

```diff
@@ -16,6 +16,7 @@ const EducationStore = () => {
 			links={ [
 				{
 					externalLink: true,
+					// eslint-disable-next-line wpcalypso/i18n-unlocalized-url
 					url: `https://wordpress.com/support/video-tutorials-add-payments-features-to-your-site-with-our-guides/`,
```

Comment:
This link doesn't seem to work. cc @emilyaudela - any idea who we can ping about it?